README.md

```diff
@@ -41,19 +41,18 @@ app.mount('#app');
 ```vue
 <template>
 	<Descope
-		:flowId.prop="'my-flow-id'"
+		flowId="'my-flow-id'"
 		@error="handleError"
 		@success="handleSuccess"
 	/>
 	<!-- additional props -->
-	<!-- :theme.prop="'dark'" theme can be "light", "dark" or "os", which auto select a theme based on the OS theme. Default is "light" -->
-	<!-- :debug.prop="true" debug can be set to true to enable debug mode -->
```

Comment:
fyi added v-bind for boolean option https://stackoverflow.com/questions/49225002/passing-boolean-vue-prop-value-in-html